osgpkitools/osg-cert-request

```diff
@@ -8,11 +8,11 @@ user.
 import urllib
 import httplib
 import sys
-import simplejson
 import os
 from optparse import OptionParser
 import subprocess
 from ssl import SSLError
+import simplejson
```

Comment:
Python 2.6 includes the `json` module in the standard library, which is a drop-in replacement for `simplejson`. Use that instead.